packages/asl-pages/pages/role/named-person-mvp/views/confirm.jsx

```diff
@@ -63,6 +63,7 @@ const Confirm = ({
 
       <div className="govuk-box requirements-box">
         {values.type === 'nacwo' && <Snippet>declarationNACWODesc</Snippet>}
+        {values.type === 'nvs' && <Snippet>declarationNVSDesc</Snippet>}
         {!mandatoryTrainingRequirementsForRoles[values.type] && (
           <Snippet>declarationOtherDesc</Snippet>
         )}
```

Comment:
There's a chance for this logic to get out of sync. `!mandatoryTrainingRequirementsForRoles[values.type]` is not checking the same thing as the other two. This also makes it harder to parse the business logic. Two options:

Put the declaration snippet ref in the mandatoryTrainingRequirementsForRoles: 
```suggestion
        <Snippet>
          {
            mandatoryTrainingRequirementsForRoles[values.type]?.declarationSnippet 
              ?? 'declarationOtherDesc'
          }
        </Snippet>
```
or use Snippet fallbacks:
```suggestion
        <Snippet fallback="declarationOtherDesc">
          {`declaration${values.type.toUpperCase()}Desc`}
        </Snippet>
```

This is possibly also highlights some other refactorings: 
* `mandatoryTrainingRequirementsForRoles` may be too specific, would `roleRequirements`, `roleSpecification` or similar be clearer?
* Snippets support nesting, so you could have a `declarations` object in content/index.js with `nacwo`, `nvs`, `default` keys then e.g. that last snippet would become 
  ```js
  <Snippet fallback="declarations.default">{`declarations.${values.type}`}</Snippet>
  ```

Reply:
other roles in upcoming declaration tickets have different content, so there are more combinations. I think, I will get more clarity once I see all declaration tickets. I thought its fine to do what I did for now and refactor on last declaration ticket. I actually added comment saying this in last PR but didn't add it here.

Reply:
but let me see how i can improve based on what i know so far